app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -888,6 +887,23 @@ impl Node {
                 <+ visualization_visible.not().and(&no_error_set);
 
 
+            // === Bounding Box ===
+
+            let visualization_size             = &model.visualization.frp.size;
+            visualization_enabled_and_visible <- visualization_enabled && visualization_visible;
```

Comment:
What does it mean that visualization is "enabled" but not "visible"? We can only show and hide visualization in GUI, so what are these 2 states about?

Reply:
This is explained in comments for the corresponding Output ports of a Node:
https://github.com/enso-org/enso/blob/f4d236fcd4c95bbe3502286fda6589313a7d72a2/app/gui/view/graph-editor/src/component/node.rs#L320-L323
Is that incorrect or inadequate? Is there some way you would suggest I should change the code to make it more clear?

Reply:
I'd only add, that visualization may be also enabled, but not visible (when the node has error).